data/actionsmap/yunohost.yml

```diff
@@ -925,6 +925,54 @@ monitor:
 
 
 #############################
+#         Settings          #
+#############################
+settings:
+    category_help: Manage YunoHost global settings
+    actions:
+
+        ### settings_list()
+        list:
+            action_help: list all entries of the settings
+            api: GET /settings
+
+        ### settings_get()
+        get:
+            action_help: get an entry value in the settings
+            api: GET /settings/<key>
+            arguments:
+                key:
+                    help: Settings key
+
+        ### settings_set()
+        set:
```

Comment:
Yeah, that's the discussion we had and I don't have strong opinions on that. Right now I would suggest starting with this simple approach, keep the settings internal, and look on how other PR brings new requirements.

And if those requirements a hook based approach, sure, let's do it, but I want to avoid YAGNI for now (you know, like those offsets :p).